docs/source/admin/capacity-planning.md

```diff
@@ -0,0 +1,246 @@
+
+An important consideration when assigning resources to users is:
+
+> What happens when users need more than I've given them?
+
+A good summary to keep in mind: **when tasks don't get enough CPU, things are slow.
+When they don't get enough memory, things are broken.**
+Which means it's very important that users have enough memory,
+but much less important that they always have exclusive access to all the CPU they can use.
+
+This relates to [Limits and Requests](limits-requests),
+because these are the consequences of your limits and/or requests not matching what users actually try to use.
+
+A table of mismatched resource allocation situations and their consequences:
+
+| issue                                                    | consequence                                                                           |
+| -------------------------------------------------------- | ------------------------------------------------------------------------------------- |
+| Requests too high                                        | Unnecessarily high cost and/or low capacity.                                          |
+| CPU limit too low                                        | Poor performance experienced by users                                                 |
+| CPU oversubscribed (too-low request + too-high limit)    | Poor performance across the system; may crash, if severe                              |
+| Memory limit too low                                     | Servers killed by Out-of-Memory Killer (OOM); lost work for users                     |
+| Memory oversubscribed (too-low request + too-high limit) | System memory exhaustion - all kinds of hangs and crashes and weird errors. Very bad. |
+
+Note that the 'oversubscribed' problem case is where the request is lower than _typical_ usage,
+meaning that the total reserved resources isn't enough for the total _actual_ consumption.
+This doesn't mean that _all_ your users exceed the request,
+just that the _limit_ gives enough room for the _average_ user to exceed the request.
+
+### Example case for oversubscribe memory
+
+Take for example, this system and sampling of user behavior:
+
+- System memory = 8G
+- memory request = 1G, limit = 3G
+- typical 'heavy' user: 2G
+- typical 'light' user: 0.5G
+
+This will assign 8 users to those 8G of RAM (remember: only requests are used for deciding when a machine is 'full').
+As long as the total of 8 users _actual_ usage is under 8G, everything is fine.
+But the _limit_ allows a total of 24G to be used,
+which would be a mess if everyone used their full limit.
+But _not_ everyone uses the full limit, which is the point!
+
+This pattern is fine if 1/8 of your users are 'heavy' because _typical_ usage will be ~0.7G,
+and your total usage will be ~5G (1 × 2 + 7 × 0.5 = 5.5).
+
+But if _50%_ of your users are 'heavy' you have a problem because that means your users will be trying to use 10G (4 × 2 + 4 × 0.5 = 10),
```

Comment:
```suggestion
But if _50%_ of your users are 'heavy' you have a problem because that means your users will be trying to use 10G (`4 × 2 + 4 × 0.5 = 10`),
```

I find it cleaner to read arithmetic in docs as inline code blocks, wdyt? (I do the same for numbers that are used often as well)